xds/internal/client/v2client_lds.go

```diff
@@ -74,9 +74,13 @@ func (v2c *v2Client) getRouteConfigNameFromListener(lis *xdspb.Listener) (string
 		}
 		return name, nil
 	case *httppb.HttpConnectionManager_RouteConfig:
-		// TODO: Add support for specifying the RouteConfiguration inline
+		// Add support for specifying the RouteConfiguration inline
```

Comment:
This is more than just return the string, it also needs to tell the client to skip RDS and call CDS.

This is a much bigger change than it looks like. I think we can leave it as is for now.

Thanks for the PR anyway.